.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*       @apoloa @gvassallo @naxhh @danipl @hredondo @otaviocarvalho @albertoolivan @carl0FF @rodolfocal
+*       @apoloa @gvassallo @naxhh @danipl @hredondo @otaviocarvalho @albertoolivan @carl0FF @rodolfocal @vetus @vetusbs
```

Comment:
why two users for the same person? vetus does not seem to exist: https://github.com/vetus